x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -190,6 +206,44 @@ export default function (ctx: FtrProviderContext) {
         await PageObjects.unifiedFieldList.expectMissingFieldListItemVisualize('bytes');
       });
 
+      it('should allow for copying the snapshot URL', async function () {
+        await PageObjects.share.clickShareTopNavButton();
+        const actualUrl = await PageObjects.share.getSharedUrl();
+        expect(actualUrl).to.contain(`?l=${DISCOVER_APP_LOCATOR}`);
+        const urlSearchParams = new URLSearchParams(actualUrl);
+        expect(JSON.parse(decompressFromBase64(urlSearchParams.get('lz')!)!)).to.eql({
+          query: {
+            language: 'kuery',
+            query: '',
+          },
+          sort: [['@timestamp', 'desc']],
+          columns: [],
+          interval: 'auto',
+          filters: [],
+          dataViewId: 'logstash-*',
+          timeRange: {
+            from: '2015-09-19T06:31:44.000Z',
+            to: '2015-09-23T18:31:44.000Z',
+          },
+          refreshInterval: {
+            value: 60000,
+            pause: true,
+          },
+        });
+        await PageObjects.share.closeShareModal();
+      });
+
+      it(`Doesn't show short urls for users without those permissions`, async () => {
```

Comment:
```suggestion
      it(`Doesn't show short urls for users without those privileges`, async () => {
```
Nit: I'd use "privileges" here for consistency.